gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.6-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.0.2-bin.zip
```

Comment:
I was under the impression that there were problems with upgrading to Gradle 8? Does it work OK now? If it doesn't and the update requires some additional changes, I'd prefer to have it in the separate PR (#2860) to not drag this one out

Reply:
Yep, but 5f66e0ddd1ef741333a73d389e5fe03d4d0fa7bd need this.